src/query_executor.c

```diff
@@ -53,6 +53,7 @@ static void _returnClause_ExpandCollapsedNodes(RedisModuleCtx *ctx, AST_Query *a
                 store = LabelStore_Get(ctx, store_type, graphName, collapsed_entity->label);
             } else {
                 /* Entity does have a label, Consult with "ALL" store. */
```

Comment:
Consider initialising graph context earlier, such that it is available at this point.